models.yml

```diff
@@ -489,6 +486,25 @@ meeting_user:
     restriction_mode: A
     to: structure_level/meeting_user_ids
 
+gender:
+  id:
+    type: number
+    restriction_mode: A
+    constant: true
+  name:
+    type: string
+    required: True
```

Comment:
Sorry, my fault, but please add an attribute `description: unique`. This may help in relational DB to create a constraint on db-level

Reply:
Done.